### PR TITLE
[Feature] Allow users to use the keyboard to get access to the main features of the LFD.

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/6423

## Description
Allow users to use the keyboard to get access to the main features of the LFD.

## Screenshots/screencasts
https://share.getcloudapp.com/z8u82qKx

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

## Notes
This PR should be merged before we merge [this one](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/416)

### CODE REVIEW

File: js/layout-javascript/agenda-code.js [hunk 1]
```diff
@@ -500,7 +500,11 @@ DynamicList.prototype.attachObservers = function() {
         });
       });
     })
-    .on('click', '.agenda-detail-overlay-close, .agenda-detail-overlay-screen', function(event) {
+    .on('click keydown', '.agenda-detail-overlay-close, .agenda-detail-overlay-screen', function(event) {
+      if (event.type !== 'click' && event.keyCode !== 32 && event.keyCode !== 13) {
```
Comment (tonytlwu) [inline js/layout-javascript/agenda-code.js:504]: Didn't we create a function in utils.js to avoid repeating this code?

Comment (YaroslavOvdii) [reply js/layout-javascript/agenda-code.js:536]: @tonytlwu Yes, moved it back.

File: js/layout-javascript/news-feed-code.js [hunk 2]
```diff
@@ -247,7 +256,11 @@ DynamicList.prototype.attachObservers = function() {
         }
       });
     })
-    .on('click', '.news-feed-detail-overlay-close, .news-feed-detail-overlay-screen', function(event) {
+    .on('click keydown', '.news-feed-detail-overlay-close, .news-feed-detail-overlay-screen', function(event) {
+      if (event.type !== 'click' && event.keyCode !== 32 && event.keyCode !== 13) {
```
Comment (tonytlwu) [inline js/layout-javascript/news-feed-code.js:260]: Why isn't this updated too?

```suggestion
      if (!_this.Utils.Event.isExecute(event)) {
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/layout-javascript/news-feed-code.js (hunk 2 above)
Comment (YaroslavOvdii) [reply js/layout-javascript/news-feed-code.js]: @tonytlwu removed

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: All click-able features in LFD
Verify: Verify all click-able features in LFD work as usual

**Pull request merged**